rosdep/base.yaml

```
@@ -460,6 +460,17 @@ castxml:
   fedora: [castxml]
   macports: [castxml]
   ubuntu: [castxml]
+catch2:
+  arch: [catch2]
```

Linter says this is available for alpine: https://pkgs.alpinelinux.org/package/edge/community/x86_64/catch2
```suggestion
  alpine: [catch2]
  arch: [catch2]
```